src/pages/index.js

```diff
@@ -73,31 +105,96 @@ function handleImageClick(cardData) {
   imageModal.open(cardData);
 }
 
+function handleLikeClick(card) {
+  api
+    .likeCard(card._id, card._isLiked)
+    .then(() => {
+      card.handleLikeClick();
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+}
+
 function handleProfileModalSubmit(inputValues) {
-  userInfo.setUserInfo(inputValues.title, inputValues.subtext);
-  //   profileTitle.textContent = inputValues.title;
-  //   profileSubtext.textContent = inputValues.subtext;
+  profileModal.setloading(true, "Saving...");
+  api
+    .setUserInfo(inputValues)
+    .then((data) => {
+      userInfo.setUserInfo(data);
+    })
+    .catch((err) => {
+      console.log(err);
+    })
+    .finally(() => {
+      profileModal.setloading(false, "Save");
+    });
+
+  userInfo.getUserInfo(inputValues.name, inputValues.about);
 }
 
-function handleNewCardModalSubmit(inputValues) {
-  debugger;
-  renderCard({ name: inputValues.title, link: inputValues.link });
+function handleNewCardModalSubmit(card) {
+  cardModal.setloading(true, "Saving...");
+  api
+    .addCard({ name: card.title, link: card.link })
+    .then((data) => {
+      renderCard(data);
+    })
+    .catch((err) => {
+      console.log(err);
+    })
+    .finally(() => {
+      cardModal.setloading(false, "Create");
+    });
   newCardModalForm.reset();
```

Comment:
`newCardModalForm.reset();`

You need to **reset** the form only  in block `then` in order not to do that immediately after `submit` because when there is an error, the modal will still be opened and the user can try to send the data again to the server without having to type the data into the inputs once more.